daemon/Cargo.toml

```diff
@@ -12,7 +12,6 @@ bytes = "1"
 chrono = { version = "0.4", features = ["serde"] }
 clap = { version = "3.0.10", features = ["derive"] }
 derivative = "2"
-derive_more = { version = "0.99.17", default-features = false, features = ["display"] }
```

Comment:
curious about this change - `derive_more` seems to be way more popular and mature? and I can't see any different options from this one? (btw: there's *so many* boilerplate `Display` manual impl, we shouldn't really bother with them unless we're doing something funky....)

Reply:
We only used it for `Display` and `FromStr` and `derive_more` doesn't have `FromStr` for enums ... :(

It is easily swapped if we want a different one!

Reply:
yeah, I know it's easily swapped, I was just curious as I suspected I missed something :) thank you for explanation!